l10n_it_account_balance_eu/models/account_balance_eu.py

```diff
@@ -0,0 +1,523 @@
+            "total_amount"
+        ]
+        self.cal_balance_ue_line_amount(balance_ue_lines, "PA")
+        self.cal_balance_ue_line_amount(balance_ue_lines, "PP")
+        balance_ue_lines["PP=A7j2"]["total_amount"] = (
+            balance_ue_lines["PA"]["rounded_amount"]
+            - balance_ue_lines["PP"]["rounded_amount"]
+        ) - (
+            balance_ue_lines["PA"]["total_amount"]
+            - balance_ue_lines["PP"]["total_amount"]
+        )
+
+        balance_ue_lines["PP=A7j2"]["rounded_amount"] = self.round_bal_val(
+            balance_ue_lines["PP=A7j2"]["total_amount"], form_data["values_precision"]
+        )
+        self.cal_balance_ue_line_amount(balance_ue_lines, "PP")
+        log_warnings = ""
+        acc_ignore = ""
+        balance_ue_lines_report_data = []
+        for line in balance_ue_lines:
+            if balance_ue_lines[line]["balance_line"].zone_bal != "--":
+                balance_ue_lines_report_data.append(
+                    {
+                        "code": balance_ue_lines[line]["balance_line"].code,
+                        "desc": balance_ue_lines[line]["balance_line"].long_desc,
+                        "amount": balance_ue_lines[line]["rounded_amount"],
+                        "accounts": balance_ue_lines[line]["account_list"],
+                    }
+                )
+            else:
+                for acc in balance_ue_lines[line]["account_list"]:
+                    if acc["amount"] != 0:
+                        acc_ignore += (
+                            "   "
+                            + acc["code"]
+                            + " "
+                            + acc["desc"]
+                            + ": "
+                            + formatLang(acc["amount"], 2)
+                            + "\n"
+                        )
+        if acc_ignore != "":
+            log_warnings += (
+                _("There are accounts to ignore but with non-zero amount:")
+                + "\n"
+                + acc_ignore
+            )
+        balance_state = "OK"
+        log_env = self.env["account.balance.eu.log"]
+        log_env.search([("balance_id", "=", form_data["id"])]).unlink()  # clear log
+        unlinked_account = []
+        tot = 0
+        self.get_account_list_amount(
+            company_id,
+            currency_id,
+            date_from,
+            date_to,
+            only_posted_move,
+            hide_acc_amount_0,
+            ignore_closing_move,
+            "non_assoc",
+            False,
+            "",
+            tot,
+            unlinked_account,
+        )
+        if (
+            balance_ue_lines["PA"]["rounded_amount"]
+            != balance_ue_lines["PP"]["rounded_amount"]
+        ):
+            balance_state = "UNBALANCED"
+            log_warnings = log_warnings + (
+                _(
+                    "NON-SQUARE Balance: {:s} (Assets) - {:s} (Liabilities) = {:s}"
```

Comment:
Partendo dal significato che è stato dato in italiano, presente nella traduzione, direi che è più "Unbalanced financial statements: ...".

Reply:
fatto come suggerito